pkg/systemd/overview-cards/realmd-operation.js

```diff
@@ -544,6 +545,7 @@ function instance(realmd, mode, realm, state) {
                             })
                             .always(function() {
                                 sub.remove();
+                                $(".realms-op-cancel").prop('disabled', false);
```

Comment:
Ok, so the button now will be there only for closing the dialog before you decide to do anything, right?

Reply:
Right, or after it failed and it shows the error message. Both cases (before clicking "join" and after failure) are covered in tests already.